src/server.js

```diff
@@ -138,20 +144,30 @@ let load = loader({
       // Create server and start listening
       let server = http.createServer(app);
       await new Promise((accept, reject) => {
-        server.listen(cfg.server.port, accept);
+        server.listen(cfg.server.port);
+        server.once('listening', () => {
+          console.log("Listening on port: " + cfg.server.port);
+        });
         server.once('error', reject);
+        server.once('close', accept);
```

Comment:
Don't do `.then(()=> process.exit(0))`... 

Promise resolution just means everything started... Ah, I see, so if you have a bug after you started...
I think what I do in queue for the backgrund processes is:
 * promise resolved for server means server is running
 * promise resolved for cron job means job is finished running

Because those are the things you care about in testing. Also as long as server is running the process keeps running (given that you don't call `process.exit`). If the promise is resolved and nothing is listening on any port, it'll shutdown (if not, then it's most likely because you have keepAlive connections to the same process which keeps it alive... or you have a setTimeout or listening on some port).

Generally node should exit when nothing is running, no need to do `process.exit` manually..  If so you would be better off figuring out why as it's bound to cause pain some day...